### PR TITLE
MC Server automatic version picker YAD prompt

### PR DESCRIPTION
**Pull request by theofficialgman**
Fabric, Forge, Vanilla, Paper, Bukkit, Spigot, Purpur, and Tuinity are ALL supported. The old system of opening the web browser and having the user choose the appropriate java version has been removed, everything is entirely automated.

this also automatically pics the version of java necessary for everything and autoselects it instead of asking the user

to close https://github.com/Botspot/pi-apps/pull/1406

### CODE REVIEW

File: apps/Minecraft Java Server/install [hunk 1]
```diff
@@ -83,16 +88,44 @@ done
...
-\nMinecraft 1.17.X should use java 16 and Minecraft 1.18+ should use java 17."
-table=("Java 17" "Java 16" "Java 8")
-userinput_func "$description" "${table[@]}"
-java_selection="$output"
+if [ -z "$server_version" ]; then
+  # tell user to choose java 8, 16, or 17
+  description="In order to run Minecraft, the game requires a specific version of Java.\
+  \nPlease choose below which version of java the server of your choice should run with.\
+  \nJava versions can be 8, 16, or 17.\
+  \n\nAll version of Minecraft before and including Minecraft 1.16.5 should use Java 8.\
```
Comment (Crilum) [inline apps/Minecraft Java Server/install:96]:
```suggestion
  \n\nAll versions of Minecraft before and including Minecraft 1.16.5 should use Java 8.\
```
Just a typo ;)

**Comment by theofficialgman** [comment]
fabric has a decent meta api, going to modify this script to include it so users don't need to use the website. don't merge yet

**Comment by theofficialgman** [comment]
🥳 ALL server types now support downloading directly through the install script
there is no more web browser opening or manual user file placing left in this script

@KolpixTV consider your suggestion fully implemented now

**Comment by rbcoder117** [comment]
@Botspot

**Comment by theofficialgman** [comment]
just rebased and cleaned up commit messages a bit. no changes.
I consider this a bugfix PR with some important feature updates. Since the MC Server script seems to be getting some use, going to go ahead and merge myself

**Pull request merged**